src/containers/category-dropdown/CategoryDropdown.tsx

```diff
@@ -99,14 +98,15 @@ const CategoryDropdown = ({
     [dispatch, fetchAllCategoriesNames]
   )
 
-  const { fetchData: handleCreateCategory } = useAxios({
-    service: createCategory,
-    defaultResponse: null,
-    fetchOnMount: false,
-    onResponse: onResponseCategory,
-    onResponseError: handleResponseError
+  const { mutate: handleCreateCategory } = useMutation({
+    mutationFn: ResourceService.createResourceCategory,
+    onSuccess: async (response) => {
+      await onResponseCategory(response)
+    },
```

Comment:
Don't add additional wrappers, add just a reference to this function
```suggestion
    onSuccess: onResponseCategory,
```